src/main/resources/db/changelog/db.changelog-RDM-10141.xml

```diff
@@ -0,0 +1,65 @@
+        CREATE INDEX IF NOT EXISTS idx_case_data_IA_appellantNameForDisplay ON public.case_data USING btree (btrim(upper((data #>> '{appellantNameForDisplay}'::text[]))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_IA_searchPostcode ON public.case_data USING btree (btrim(upper((data #>> '{searchPostcode}'::text[]))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_IA_legalRepReferenceNumber ON public.case_data USING btree (btrim(upper((data #>> '{legalRepReferenceNumber}'::text[]))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_IA_homeOfficeReferenceNumber ON public.case_data USING btree (btrim(upper((data #>> '{homeOfficeReferenceNumber}'::text[]))));
+
+        CREATE INDEX IF NOT EXISTS idx_case_data_sscs_appellant_dob ON public.case_data USING btree (btrim(upper((data #>> '{appeal,appellant,identity,dob}'::text[]))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_sscs_attachToCaseReference ON public.case_data USING btree (btrim(upper((data #>> '{attachToCaseReference}'::text[]))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_sscs_dateSentToDwp ON public.case_data USING btree (btrim(upper((data #>> '{dateSentToDwp}'::text[]))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_sscs_directionDueDate ON public.case_data USING btree (btrim(upper((data #>> '{directionDueDate}'::text[]))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_sscs_appeal_hearingType ON public.case_data USING btree (btrim(upper((data #>> '{appeal,hearingType}'::text[]))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_sscs_surname ON public.case_data USING btree (btrim(upper((data #>> '{surname}'::text[]))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_sscs_urgentCase ON public.case_data USING btree (btrim(upper((data #>> '{urgentCase}'::text[]))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_sscs_createdInGapsFrom ON public.case_data USING btree (btrim(upper((data #>> '{createdInGapsFrom}'::text[]))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_last_state_modified_date ON case_data (last_state_modified_date);
+
+        CREATE INDEX IF NOT EXISTS idx_case_data_sscs_subscriptionsjp_tya ON public.case_data USING btree (btrim(upper((data #>> '{subscriptions,jointPartySubscription,tya}'::text[]))));
+
+        CREATE INDEX IF NOT EXISTS idx_case_data_last_state_modified_date ON case_data (last_state_modified_date);
+        CREATE INDEX IF NOT EXISTS idx_case_user_user_id ON case_users (user_id);
+
+        DROP INDEX IF EXISTS idx_case_data_generated_surname;
+        DROP INDEX IF EXISTS idx_case_data_welsh_dnoutcomecase_solhowtopay_applyforda  ;
+        DROP INDEX IF EXISTS idx_case_data_welsh_dnout_d8_sol_apply;
+        DROP INDEX IF EXISTS idx_case_data_interlocreferraldate;
+        DROP INDEX IF EXISTS idx_case_data_isscottishcase ;
+        DROP INDEX IF EXISTS idx_case_data_application_type;
```

Comment:
Did we check dropping of these indexes with respective service teams?
At least we should inform that to find out why these indexes are not in actual use though they once have got them created.